lib/grdefs.h

```diff
@@ -82,9 +83,9 @@ static const ddgr_color GR_NULL = 0xffffffff, // don't do a thing with this.
 #define GR_COLOR_CHAR 1 // ASCII 1 and (r,g,b) changes current text color in string.
 
 //	MACROS
-static inline ddgr_color GR_RGB(int r, int g, int b) { return ((r << 16) + (g << 8) + b); }
+static inline ddgr_color GR_RGB(uint8_t r, uint8_t g, uint8_t b) { return ((r << 16) + (g << 8) + b); }
 
-static inline uint16_t GR_RGB16(int r, int g, int b) { return (((r >> 3) << 10) + ((g >> 3) << 5) + (b >> 3)); }
+static inline uint16_t GR_RGB16(uint8_t r, uint8_t g, uint8_t b) { return (((r >> 3) << 10) + ((g >> 3) << 5) + (b >> 3)); }
```

Comment:
Probably is better change r,g,b to `uint32_t`? All calls defines r,g,b as `int`.

Reply:
Is there any changes are planned?

Reply:
(Not sure where Github lost my comment, but here goes again: )

Changing to `uint32_t` is a *big* problem. The callsite effectively uses `char b = 255; GR_RGB16(r,g,b);`, which causes b=0xffffffff, and all those blue bits spill over into `r` and `g`, making the result of GR_RGB useless.
So no change planned, uint8_t is just right. (A function probably should not take types whose range is not handled anyway.)